.eslintrc.cypress.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
You shouldn't have three of these files, one file called `.eslintrc.json` with two override blocks, one for jest and one for cypress. Only by getting this file right will the errors go away.

Reply:
Take a look around here for some inspiration: https://github.com/NoroffFEU/workflow/branches